src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -41,6 +44,23 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives {
               }
             }
           }
+        } ~
+        pathPrefix("policy") {
+          pathPrefix(Segment) { resourceTypeName =>
+            pathPrefix(Segment) { resourceId =>
+              pathPrefix(Segment) { accessPolicyName =>
```

Comment:
you can do
`path(Segment / Segment / Segment) { (resourceTypeName, resourceId, accessPolicyName) =>`

Reply:
I am not sure what we want to call this, but just a post to /api/google/policy/rt/r/p seems to vague. Maybe synchronize?